Auth0/Auth0Authentication.swift

```diff
@@ -476,6 +476,10 @@ struct Auth0Authentication: Authentication {
                        telemetry: self.telemetry)
     }
 
+    func customTokenExchange(subjectTokenType: String, subjectToken: String, scope: String, audience: String? = nil) -> Request<Credentials, AuthenticationError> {
```

Comment:
`audience` should not have a default value here (`audience: String? = nil`). Default values for params go in the respective extension method in Authentication.swift

Reply:
I mean other methods like `renew` do it. However, will fix

Reply:
That's probably a leftover from years ago.